In [1]:
import numpy as np
import pandas as pd

# Range Rule of Thumb 
- they are used to find the quickly rough estimation of standard deviation, when you have no builten formula of standard deviation and data values should be unimodal
- they are close to standard deviation but they are not exact std
- formula as (range/4)

In [32]:
# Same Data of Coefficient variation 
speed=np.array([112.5,15.0,200.0,123.2,142.6,190.23,20.0,10.0,23.0,200.0]) # KMPH

In [33]:
(np.max(speed)-np.min(speed))/4

47.5

In [34]:
np.std(speed)

76.27008509894294

- In range rule of thumb the standard devitaion is 47.5 which is near to real standard deviation so its mean the RRT shows approximate std not real